src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx

```diff
@@ -90,6 +90,17 @@ When you edit the site, include the Jira issue key (DOC-1234, for example) in yo
       </td>
     </tr>
 
+        <tr>
+      <td>
+        User story
+      </td>
+
+      <td>
+        *According to [Atlassian](https://www.atlassian.com/agile/project-management/user-stories) "A user story is an informal, general explanation of a software feature written from the perspective of the end user. Its purpose is to articulate how a software feature will provide value to the customer. 
+        *User stories are often expressed in a simple sentence, structured as follows: 'As a [persona], I [want to], [so that].'"'
```

Comment:
* Is this continuing the quote from before? If so, I think you probably want a separate set of quotes. 
* This closes with `'"'` I think at least one of those quoets might be extraneous?